resources/qml/MessageInput.qml

```diff
@@ -335,6 +335,7 @@ Rectangle {
                     y: messageInput.positionToRectangle(messageInput.completerTriggeredAt).y - height
 
                     enter: Transition {
+                        enabled: !Settings.reducedMotion
```

Comment:
Originally we were told that fade effects are fine for reducedMotion, which is why we didn't disable those. Do you have a reason to do otherwise?

Reply:
No, I mechanically disabled _all_ transitions when reducedMotion is enabled. Are there any specific instances where you'd like to keep these? My motivation for disabling transitions is poor performance - on nvidia-wayland, the transitions stutter sometimes on a fairly powerful gpu (1060).

Reply:
We have the reduced motion setting not because of stutter, we have them to provide a usable experience to people, who get dizzy when stuff on their screen moves. That's why we use fade animations instead. The stutter could probably be easily fixed by figuring out why it stutters, it is very unlikely to be related to your GPU struggling, but probably more because Nheko blocks the UI thread with database work or so.